app.rb

```diff
@@ -1,8 +1,17 @@
 require 'sinatra/base'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :peeps
+  end
+
+  post '/send-peep' do
```

Comment:
If you rename `'/send-peep'` to `'/create-peep'` you should see what you're expecting.